numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -251,120 +284,196 @@ def get_system_ctk(*subdirs):
         result = os.path.join("/usr/local/cuda", *subdirs)
         if os.path.exists(result):
             return result
+        return None
+    return None
 
 
-def get_conda_ctk():
```

Comment:
this is technically public facing, though likely used minimally, do we want to add some kind of deprecation warning around this?